src/common/utils/oidc/secret.go

```diff
@@ -116,15 +109,22 @@ func (dm *defaultManager) VerifySecret(ctx context.Context, userID int, secret s
 	if err != nil {
 		return verifyError(err)
 	}
-	err = dm.SetSecret(oidcUser.UserID, secret, t)
+	tb, _ := json.Marshal(t)
```

Comment:
what's the reason to eat these errors? At least we can have logs.

Reply:
It returns error when the `t` can not be encoded to json, which is not true in this case, so it's ok to eat the error.
Anyway there's no harm add a log